crates/octez/src/async/bootstrap.rs

```diff
@@ -91,16 +95,127 @@ impl From<&BootstrapAccounts> for Value {
     }
 }
 
+#[serde_with::serde_as]
+#[serde_with::skip_serializing_none]
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+pub struct BootstrapContract {
+    /// Smart contract code in micheline format as a `serde_json::Value` instance.
```

Comment:
Thanks, perhaps it would be helpful to be explicit that the code does not get validated.

Reply:
Updated